### ISSUE TITLE
training loss

### ISSUE THREAD

**Issue by Albert-Ma** [google-research/pegasus#60]
Hi,

Did you just add the two loss  together e.g. gsg loss and mlm loss, 

Loss = L1+L2,

Did you use some tricks to balance the two loss?
@JingqingZ  Thank you

**Comment by JingqingZ** [comment]
Yes, and no tricks. the model (MLM & Ind-Orig) reported in Fig 4 (a) in our paper is simply adding the two loss functions. But please note, in the final model PEGASUS_LARGE, there is no MLM and only GSG is applied.

**Comment by Albert-Ma** [comment]
Ok, thanks
